api/services/taxonomyService.ts

```diff
@@ -1,31 +1,35 @@
+import { NotFoundError } from 'typescript-rest/dist/server/model/errors';
 import { makeSCOSRequest } from './scosService';
 import getLogger from '../utils/logger';
 import TaxonomyDto from '../models/dto/taxonomyDto';
 import ScosSubcategoryDto from '../models/scosApi/scosSubcategoryDto';
 import ScosCategoryDto from '../models/scosApi/scosCategoryDto';
 import { CATEGORIES_TABLE } from '../utils/constants';
-import { NotFoundError } from 'typescript-rest/dist/server/model/errors';
 
 const log = getLogger('taxonomyService');
 
 async function getSubcategories(id: string): Promise<TaxonomyDto[]> {
   log.debug(`Getting subcategories of category id ${id}`);
 
   const query = `SELECT subcat FROM ${CATEGORIES_TABLE} WHERE categoryid = '${id}'`;
-  const response: Pick<ScosCategoryDto, 'subcat'>[] = await makeSCOSRequest(query);
+  const response: Pick<ScosCategoryDto, 'subcat'>[] = await makeSCOSRequest(
+    query
+  );
 
   if (response?.length === 0) {
     throw new NotFoundError(`Invalid category id ${id}`);
   }
 
-  return response[0].subcat.map((subcat: ScosSubcategoryDto): TaxonomyDto => ({
-    id: subcat.subcategoryid,
-    description: subcat.subcategory,
-    parentCategoryId: id,
-    // TODO: hardcoded bc I'm not sure if this is still available in the data
-    level: 1,
-  }
-  ));
+  return response[0].subcat.map(
+    (subcat: ScosSubcategoryDto): TaxonomyDto => ({
+      // ids aren't available in the agencies table, so we're just passing around the string for now
+      id: subcat.subcategory,
+      description: subcat.subcategory,
+      parentCategoryId: id,
+      // "categories" are level 1; "subcategories" are level 2
```

Comment:
Levels only apply to subcategories - the UI only shows the first 5 items in the subcategories. Since we also sort by the `level` it is showing the most important items first. I am going to look into the data set to see where this lives in the new version.